### ISSUE TITLE
Clean up the 'git-for-windows' org

### ISSUE THREAD

**Issue by dscho** [git-for-windows/git#53]
As of this morning, there were four repositories [in the *git-for-windows* org](https://github.com/git-for-windows) that leave room for confusion with the current push for an MSys2-based Git for Windows:

* [x] mgwport
* [X] pacman-repository
* [x] sdk
* [x] sdk-packages

I already moved the `pacman-repository` repository into my personal GitHub space because its functionality is superseded by our [Pacman repository on Bintray](https://bintray.com/git-for-windows/pacman).

We should move the other three packages out of the way, too, lest some developer thinks that we are still basing Git for Windows 2.x on MSys 1.x.

**Comment by sschuberth** [comment]
It's a bit unfortunate that we actually have to do this as GitHub does not offer a way to group repositories per project. So I plan to move sdk and sdk-packages back to my personal account again. @t-b do you want me to move mgwport there, too, to keep all the MSYS1-based stuff in one place, or do you want to take it?

**Comment by t-b** [comment]
@sschuberth Moving mgwport et. al. to your account is fine with me.

**Comment by sschuberth** [comment]
It *totally* annoys me again that GitHub does not allow me to group my user's repositories by project / topic. Moving the mgwport, sdk and sdk-packages repos to my user would mix with my existing repos of other projects, and I really do not like that as I'm starting to lose the overview.

The only option seems to be create another org, and move the repos there. So, any objections that I create a new git-for-windows-msys1 (or so) org?

I'm also totally open to moving away from GitHub, but unfortunately neither [Bitbucket](https://bitbucket.org/site/master/issue/2323/create-a-way-to-group-repositories) nor GitLab have that feature either ([GitLab has groups](https://about.gitlab.com/2014/06/30/feature-highlight-groups/), but they work different to what I'd expect; they're merely just an alias for teams / orgs). Any other hosting suggestions?

**Comment by dscho** [comment]
Why not just move the repositories to the `msysgit` org (it houses our MSys1-based Git stuff, after all), and be done with this issue?

**Comment by dscho** [comment]
@sschuberth if you do not want to move these three repositories to `msysgit`, I could also move them to my user realm. IMO this ticket is not important enough to worry about. There are substantially more important tickets that we still need to address urgently.

**Comment by sschuberth** [comment]
> Why not just move the repositories to the msysgit org

Because the whole point of my "Git for Windows SDK" effort was to make clear it's a *different* project from msysgit, with a new installer for the development environment.

> I could also move them to my user realm

No thanks. I'd like to have these repos under my control as I'm still planning to occasionally work on them. It's still my pet project after all.

**Comment by dscho** [comment]
> the whole point of my "Git for Windows SDK" effort was to make clear it's a different project from msysgit

But the Git for Windows SDK effort was *based* on msysgit. I think it is a little bit disrespectful to say otherwise.

> It's still my pet project after all.

Fine with me. I just wanted to help you get to the more interesting tickets. I do not really have a strong opinion where the repositories in question end up. I just do not want to spend more time on this ticket because I do not see enough value in it to spend more than five minutes on it. I have bigger fish to fry, after all ;-)

**Comment by dscho** [comment]
So if you do not want to move them, we can just change the description. It would mean that way too many developers would still stumble over them and mistake them for the Git for Windows SDK that we use now, but if that is what you want, I won't try to change your mind anymore.

**Comment by sschuberth** [comment]
> But the Git for Windows SDK effort was based on msysgit. I think it is a little bit disrespectful to say otherwise.

Without meaning to be disrespectful, the Git for Windows SDK effort was pretty much a rewrite of msysgit. Measured by the amount of work put into it, it could as well have been MSYS2-based. Doing that would have taken me the same amount of time. I just did decide against MSYS2 because at the time I was starting the Git for Windows SDK it was not yet mature enough for my taste.

> So if you do not want to move them, we can just change the description.

In case you didn't notice, I did that already quite some time ago. All project descriptions explicitly mention "MSYS1" to differentiate from the current MSYS2 efforts.

And like I said, I'm fine with moving them, I was just awaiting your answer to my question about creating an git-for-windows-msys1-org. 

Anyway, I'm feeling pushed to move the repos by you repeatedly say that devs would confuse them with being part of the current MSYS2 efforts, so I'll move them to my user ASAP, and probably move them again later.
